### DISCUSSION TITLE
Canal Telecinco de España.

### DISCUSSION THREAD

**Discussion by emege1** [iptv-org/community#1045] [Q&A]
Se podrá incluir el canal Telecinco de España en la lista m3u? Está ya disponible desde hace poco tiempo el canal Cuatro que es también del grupo MediaSet.
Muchas gracias.
Saludos

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/es.m3u#L513-L514
